src/main/java/javax/security/enterprise/authentication/mechanism/http/LoginToContinue.java

```diff
@@ -51,7 +51,7 @@
 import javax.resource.spi.AuthenticationMechanism;
 
 /**
- * The <code>LoginToContinue</code> annotation provides an application the ability to declaratively 
+ * The <code>LoginToContinue</code> annotation provides an application the ability to declarative
```

Comment:
This change actually seems wrong -- shouldn't it be "declaratively" -- an adverb modifying the verb "add -- rather than "declarative", an adjective modifying ... what?